CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+This project is open to contributors, and we'd like to make it as easy as possible to share your ideas, and your code.
+
+Firstly, no contribution is too small. If you see something that's not right, please feel free to submit a spontaneous PR. It will very likely be reviewed same day.
+
+## Where to start
+
+If you don't know where to start, have a look at the list of [issues](https://github.com/tomarrell/lbadd/issues). If you find something interesting, feel free to say so in a comment, and you can be assigned to the issue.
+
+If someone is already assigned to an issue, feel free to mention that you're interested as well. Most issues can likely be broken down into more fine-grained tasks for newcomers to contribute to.
+
+If you don't find something in the issues, don't worry! There is **plenty** that is *likely not listed there*. If you think you have a clear idea about something that should be done, feel free to open an issue yourself and start a discussion. A maintainer will get back to you, and most likely assign you to the task or clarify some requirements beforehand.
+
+## Project Setup
+
+In order to begin contributing, please fork the project to your personal Github account. You can then clone the project to your machine, and begin work on a new branch.
+
+You'll need to have installed:
+
+- Go compiler `version >=1.13`
+- golint `master // TODO`
+- errcheck `master // TODO`
+- gosec `master // TODO`
+
+## Contributing
+
+On this separate branch, make the minimum set of changes that are required to fulfill the task. If possible, please also write clear, package prefixed commit messages in order to make reviews easier. Once you're happy with the changes you've made, please make sure that you've added tests, and that the existing tests pass by running. We aim for high test coverage, so you'll likely be asked to add tests around areas that you've missed.
+
+```bash
+$ make test
+```
+
+This will verify that you're not causing any regressions in functionality.
+
+You'll also need to run the linters. The linters that are run are listed above under **Project Setup**.
+
+```bash
+$ make lint
```

Comment:
Can we modify the makefile to `go get` the tools? That will basically be a no-op if they are already installed and download them otherwise.

Or should we introduce a `tools.go` file? If you think that this needs more discussion, I'll open a ticket where we can discuss pros and cons.

Reply:
Yeah this would be useful. A ticket for this sounds good.

Reply:
https://github.com/tomarrell/lbadd/issues/27